openassessment/xblock/static/js/spec/lms/oa_staff_area.js

```diff
@@ -643,9 +643,9 @@ describe('OpenAssessment.StaffAreaView', function() {
             $assessment = getAssessment(staffArea, staffAreaTab);
             $submitButtons = $('.action--submit', $assessment);
             expect($submitButtons.length).toBe(2);
-            expect($submitButtons).toHaveClass('is--disabled');
+            expect($submitButtons).toHaveAttr('disabled');
```

Comment:
Since this test is specifically checking 2 different buttons, I changed it to look directly at the disabled attribute instead of using helper methods.